### ISSUE TITLE
No rule to make target 'src\vm\jvm\runtime\org\perl6\nqp\io\*.java'

### ISSUE THREAD

**Issue by patrickbkr** [Raku/nqp#629]
When building Rakudo on Azure Pipelines the NQP build fails on JVM / Windows with the following error:

```
gmake: *** No rule to make target 'src\vm\jvm\runtime\org\perl6\nqp\io\*.java', needed by 'gen\jvm\share\runtime\nqp-runtime.jar'.  Stop.
```

The full build output can be found here: <
https://dev.azure.com/patrickboeker/Precomp-Builds/_build/results?buildId=27&view=logs&j=d117b27c-2a1d-5196-225b-90666a1f0956&t=a257d076-54e3-5e77-52c6-5b14534967d2>

**Comment by vrurg** [comment]
Slashes vs. backslashes. gmake can't deal with `\`.

**Comment by vrurg** [comment]
Basically, there was a good idea of throwing away gmake support on Windows altogether.

**Comment by patrickbkr** [comment]
This is now fixed with [adfa192bc](https://github.com/rakudo/rakudo/commit/adfa192bc187fc4368e1323774f9c1af95c22aa4). This is one of the many the reasons why I have a bit of a dislike for Windows.
